test/cypress/e2e/search.spec.js

```diff
@@ -103,15 +88,17 @@ describe('Search', () => {
    */
   it('Search for Delegate using keyboard Enter, signed off', () => {
     cy.visit(urls.dashboard);
-    cy.get(ss.searchInput).click().type(`${accounts['mainnet delegate'].username}{enter}`);
-    assertDelegatePage(accounts['mainnet delegate'].username, accounts['mainnet delegate'].address);
+    cy.get(ss.searchIcon).click();
+    cy.get(ss.searchInput).type(`${accounts['mainnet delegate'].username}{enter}`);
+    cy.get(ss.searchDelegatesRow).find(ss.delegateName).eq(0).should('have.text', accounts['mainnet delegate'].username);
```

Comment:
same. you are testing the result of Enter here, not the content of the search bar. The content of the search bar should be tested in unit tests